wire-ios-data-model/Tests/Source/Model/Messages/ZMAssetClientMessageTests.swift

```diff
@@ -249,7 +249,7 @@ extension ZMAssetClientMessageTests {
 
         let otrKey = Data.randomEncryptionKey()
         let encryptedData = data.zmEncryptPrefixingPlainTextIV(key: otrKey)
-        let sha256 = encryptedData.zmSHA256Digest()
+        let sha256 = encryptedData!.zmSHA256Digest()
```

Comment:
maybe better to safely unwrap the value and do a XCTFail() in order not to crash the tests

```suggestion
        guard let encryptedData = encryptedData else {
          XCTFail("expected encrypted data") 
          return 
        }
        let sha256 = encryptedData.zmSHA256Digest()
```

Reply:
Done